data/tribes/atlanteans.lua

```diff
@@ -334,64 +334,95 @@ tribes:new_tribe {
    },
 
    ship_names = {
-      "Abaco",
-      "Agate",
-      "Alexandrite",
-      "Amber",
-      "Amethyst",
-      "Anguilla",
-      "Antigua",
-      "Aquamarine",
-      "Atlantean's Stronghold",
-      "Atlantis",
-      "Bahama",
-      "Barbados",
-      "Barbuda",
-      "Beryl",
-      "Blanquilla",
-      "Caicos",
-      "Cassiterite",
-      "Cat's Eye",
-      "Citrine",
-      "Coliondor",
-      "Dominica",
-      "Eleuthera",
-      "Emerald",
-      "Grenada",
-      "Guadelope",
-      "Inagua",
-      "Jundlina",
-      "Juventud",
-      "King Ajanthul",
-      "King Askandor",
-      "Kitts",
-      "Loftomor",
-      "Malachite",
-      "Martinique",
-      "Montserrat",
-      "Moonstone",
-      "Mystic Quartz",
-      "Nassau",
-      "Nevis",
-      "Obsidian",
-      "Onyx",
-      "Opol",
-      "Orchila",
-      "Ostur",
-      "Pearl",
-      "Sapphire",
-      "Satul",
-      "Sidolus",
-      "Sphalerite",
-      "Spider",
-      "Spinel",
-      "Sunstone",
-      "Tiger Eye",
-      "Tobago",
-      "Topaz",
-      "Tortuga",
-      "Tourmaline",
-      "Trinidad",
+      pgettext("shipname", "Abaco"),
+      -- TRANSLATORS: This Atlantean ship is named after a mineral
+      pgettext("shipname", "Agate"),
+      -- TRANSLATORS: This Atlantean ship is named after a mineral
+      pgettext("shipname", "Alexandrite"),
+      -- TRANSLATORS: This Atlantean ship is named after a mineral
+      pgettext("shipname", "Amber"),
+      -- TRANSLATORS: This Atlantean ship is named after a mineral
+      pgettext("shipname", "Amethyst"),
+      pgettext("shipname", "Anguilla"),
+      pgettext("shipname", "Antigua"),
```

Comment:
Shall we spend a Translators comment for the Islands as basis for names as well?

Reply:
I'll add them if you tell me which atlantean ships are named after islands, my geographical knowledge of exotic countries is … modest ;)

Reply:
Well as far as I know it here is the list:
Abaco
Anguilla
Antigua
Bahama
Barbados
Barbuda
Blanquilla
Caicos
Dominica
Eleuthera
Grenada
Guadeloupe (there is a typo in it)
Inagua
Kitts
Martinique
Montserrat
nassau
Nevis
Orchila
Tobago
Tortuga
Trinidad

Reply:
Thanks, added the comments and fixed the typo :)